### PR TITLE
Only one proc to activate away missions

### PR DESCRIPTION
**Pull request by ShiftyRail**
Which is prosaically called `admin_active()`

### CODE REVIEW

**Comment by Kurfursten** [comment]
Now make it so this is called if you click on the center as an adminghost

**Comment by ShiftyRail** [comment]
fair

File: code/modules/awaymissions/gateway.dm [hunk 1]
```diff
@@ -266,7 +272,10 @@ obj/machinery/gateway/centerstation/process()
 			calibrated = 1
 			return
 
-/obj/machinery/gateway/centerstation/attack_ghost(mob/user as mob)
+/obj/machinery/gateway/centerstation/attack_ghost(mob/user)
```
Comment (Kurfursten) [inline code/modules/awaymissions/gateway.dm:275]: did you test this? pretty sure aghosts call attack_ai when they click things, not attack_ghost

Comment (ShiftyRail) [reply code/modules/awaymissions/gateway.dm:275]: I have now and I can confirm it works

**Pull request merged**